common/lib/xmodule/xmodule/lti_2_util.py

```diff
@@ -170,7 +170,7 @@ def _lti_2_0_result_get_handler(self, request, real_user):
             "@context": "http://purl.imsglobal.org/ctx/lis/v2/Result",
             "@type": "Result"
         }
-        self.system.rebind_noauth_module_to_user(self, real_user)
+        self.system.service(self, 'rebinder').rebind_noauth_module_to_user(self, real_user)
```

Comment:
Nit: when we're modifying a line, let's replace `system` with `runtime`, as the latter is [deprecated](https://github.com/openedx/edx-platform/blob/16fa97dde4560edb347a3743a9e7e345a6d74430/openedx/core/djangoapps/xblock/runtime/shims.py#L390). This applies to the whole PR.